yoc/yoc-htb.js

```diff
@@ -178,6 +182,19 @@ function YocHtb(configs) {
         queryObj.cb
             = 'window.' + SpaceCamp.NAMESPACE + '.' + __profile.namespace + '.adResponseCallbacks.' + callbackId;
 
+        if (identityData && identityData.AdserverOrgIp && identityData.AdserverOrgIp.data) {
+            var adsrvrUids = identityData.AdserverOrgIp.data.uids;
+            if (Utilities.isArray(adsrvrUids)) {
+                for (var j = 0; j < adsrvrUids.length; j++) {
+                    if (adsrvrUids[j].ext && adsrvrUids[j].ext.rtiPartner === 'TDID') {
+                        queryObj.tdid = adsrvrUids[j].id;
```

Comment:
Missing system tests to validate request contains `AdserverOrgIp` data. Please refer to the following doc for more information: https://kb.indexexchange.com/ix_library_partners/build_adapter/test_case_reference.htm#validateBidRequestWithUAdSrvrOrg

Reply:
Test has been added

Reply:
Test had been added